packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -15,17 +15,30 @@ import ArbitrumNovaLogo from '../assets/ArbitrumNovaLogo.webp'
 
 const INFURA_KEY = process.env.REACT_APP_INFURA_KEY as string
 
+const MAINNET_INFURA_RPC_URL = `https://mainnet.infura.io/v3/${INFURA_KEY}`
```

Comment:
in this file, Rinkeby is being referenced by 2 helper functions - `getNetworkName` and `getNetworkLogo`. Are we sure we want to keep them?